doc/source/dev/conduct/report_handling_manual.rst

```diff
@@ -0,0 +1,172 @@
+.. _CoC_reporting_manual:
```

Comment:
I think this tag is wrong?

Reply:
Don't think so, cross-link worked correctly on a local doc build. If I'm missing something you have to be a bit more precise in pointing out the problem.